packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -102,9 +117,18 @@ const FolderTreeItem = ({
         <span style={{ paddingLeft: `${level}rem` }}>
           <Arrange
             fitStart={(
-              <Spacer paddingLeft="medium" paddingRight="medium" isInlineBlock>
-                {itemIcon}
-              </Spacer>
+              <>
+                <Spacer paddingLeft="medium" paddingRight="medium" isInlineBlock>
+                  {
+                    isFolder ? (
+                      <Spacer paddingRight="small" isInlineBlock>
+                        {expandCollapseIcon}
```

Comment:
Should this icon be center aligned vertically?

Reply:
Oddly the default [vertical-align property](https://github.com/cerner/terra-core/blob/main/packages/terra-icon/src/Icon.module.scss#L7) from `terra-icon` looks fine for default-sized icons, just not the 8px x 8px caret icons. Overriding the style and setting it to `baseline` seems to center them properly.

[a45606e](https://github.com/cerner/terra-framework/pull/1915/commits/a45606ee7c783e477d67e17b25c62af7ac9d7678)